app/assets/scripts/components/share.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 import React from 'react';
 import url from 'url';
+import hideOnBlur from 'react-click-outside';
```

Comment:
@dereklieu Yes, that was my mistake (did read). I actually made two versions of this commit. In the first one, I made the indicator button into a React component solely to wrap it with `react-click-outside`, but I decided that was overcomplicated so I switched to the `onMouseDown` hack (and also forgot to implement the share button properly). If you think that using this extension may be more mobile-friendly than the method I'm using for the indicators button, I can revert to the commit where I had made it into a React component.